crypto/stark-verifier-ethereum/contracts/hashing.sol

```diff
@@ -10,4 +12,8 @@ contract Hashing {
     function double_width_hash(bytes32 preimage_a, bytes32 preimage_b) internal pure returns(bytes32) {
         return hasher(abi.encodePacked(preimage_a, preimage_b));
     }
+
+    function double_width_masked_hash(bytes32 preimage_a, bytes32 preimage_b) internal pure returns(bytes32) {
```

Comment:
The mask is part of the definition of our hash function, so it should not be exposed in the API of a hash-function abstraction.

If you need both masked and unmasked hash functions (I don't think we do?) then these are two different hash functions, and should have their own abstractions (which is awkward to do in Solidity, but better than a leaking abstraction)